mne/io/edf/edf.py

```diff
@@ -20,6 +20,49 @@
 from ..constants import FIFF
 from ...filter import resample
 from ...externals.six.moves import zip
+from ...utils import copy_function_doc_to_method_doc
+
+
+def get_edf_events(raw):
```

Comment:
I made this into a function so that it actually gets documented (since we don't document the methods).